ci/jenkins/Jenkins_steps.groovy

```diff
@@ -862,6 +862,21 @@ def test_unix_clojure_cpu() {
     }]
 }
 
+
+def test_unix_clojure_cpu_nightly() {
+    return ['Clojure: CPU': {
+      node(NODE_LINUX_CPU) {
```

Comment:
This content should be the same as in tests/nightly/Jenkinsfile

Sorry that I didn't have the chance to bring tests/nightly/Jenkinsfile to the new format, but until then, please just write the same thing in both locations until they've been consolidated.